domain/src/main/java/org/oppia/android/domain/state/BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+"""
+build bazel for domain state
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "state_deck",
+    srcs = [
+        "StateDeck.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//model:exploration_java_proto_lite"
+    ]
```

Comment:
Add all missing deps that are being highlighted in the build error. 
Ditto for other added libraries.

Reply:
@prayutsu  that I have understood but the main problem is how to find which proto is to be added
like AnswersAndResponse.java is being used in StateDeck but now how can I find the particular of this in bazel/model

Reply:
You need to find the proto file that contains the message `AnswerAndResponse` in it and then you can specify the library of that file here.
To find the .proto file just press `Ctrl + Shift + R` and type the keywords that you want to search in the whole codebase.